src/python/createrepo_c/__init__.py

```diff
@@ -326,3 +330,48 @@ def decompress_file(src, dst, comtype, stat=None):
 compression_suffix  = _createrepo_c.compression_suffix
 detect_compression  = _createrepo_c.detect_compression
 compression_type    = _createrepo_c.compression_type
+
+
+# If we have been built as a Python package, e.g. "setup.py", this is
+# where the binaries will be located.
+_DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+
+# Where we will look for the binaries. Default to looking on the system PATH.
+# If one of the following test succeeds, we can change this to look somewhere else.
+# Otherwise, probably, we were not built as a Python package, e.g. RPM, "cmake ..; make"
+# In that case, let's just assume that the binary will be on the PATH somewhere.
+_BIN_DIR = ""
+
+# Support running tests from the source tree
+if not os.path.exists(_DATA_DIR):
+    # We can't find it. This might mean we've been built as an "editable" Python installation
+    # Try building the path manually.
+    try:
+        from skbuild.constants import CMAKE_INSTALL_DIR as SKBUILD_CMAKE_INSTALL_DIR
+        _data = os.path.abspath(os.path.join(
+            os.path.dirname(__file__), '..', SKBUILD_CMAKE_INSTALL_DIR, 'src/python/createrepo_c/data'))
+        if os.path.exists(_data):
+           _BIN_DIR = os.path.join(_data, 'bin')
+    except ImportError:
+        # Scikit isn't available
+        pass
```

Comment:
Need also to add 
```
else:
    _BIN_DIR = os.path.join(_DATA_DIR, 'bin')
```

Otherwise trying to run any of the executable after installing the wheel ends up in an infinite loop.